js/main.js

```diff
@@ -0,0 +1,77 @@
+// eslint-disable-next-line no-unused-vars
+function changeIcon(icon) {
+  icon.classList.toggle('fa-times');
+}
+
+const navLinks = document.querySelectorAll('.nav-item');
+const menuToggle = document.getElementById('collapsibleNavbar');
+// eslint-disable-next-line no-undef
+const bsCollapse = new bootstrap.Collapse(menuToggle);
+navLinks.forEach((n) => {
+  n.addEventListener('click', () => { bsCollapse.toggle(); });
+});
+
+const featuredSpeakers = [{
+  id: 1,
+  img: 'farhan.svg',
```

Comment:
[OPTIONAL] Please fix the broken speaker image. ↓


<img width="1255" alt="Screenshot 2023-01-07 at 6 22 33 PM" src="https://user-images.githubusercontent.com/25887253/211158160-13a0916e-4c14-4260-a991-cf1ace3676c2.png">